models/aeif_cond_alpha_multisynapse.cpp

```diff
@@ -318,40 +341,29 @@ aeif_cond_alpha_multisynapse::State_::set( const DictionaryDatum& d )
 {
   updateValue< double >( d, names::V_m, y_[ V_M ] );
 
-  if ( ( d->known( names::g_ex ) ) && ( d->known( names::dg_ex ) )
-    && ( d->known( names::g_in ) ) && ( d->known( names::dg_in ) ) )
+  if ( ( d->known( names::dg ) ) && ( d->known( names::g ) ) )
   {
-    const std::vector< double > g_exc =
-      getValue< std::vector< double > >( d->lookup( names::g_ex ) );
-    const std::vector< double > dg_exc =
-      getValue< std::vector< double > >( d->lookup( names::dg_ex ) );
-    const std::vector< double > g_inh =
-      getValue< std::vector< double > >( d->lookup( names::g_in ) );
-    const std::vector< double > dg_inh =
-      getValue< std::vector< double > >( d->lookup( names::dg_in ) );
-
-    if ( ( g_exc.size() != dg_exc.size() ) || ( g_exc.size() != g_inh.size() )
-      || ( g_exc.size() != dg_inh.size() ) )
+    const std::vector< double > dg =
+      getValue< std::vector< double > >( d->lookup( names::dg ) );
+    const std::vector< double > g =
+      getValue< std::vector< double > >( d->lookup( names::g ) );
+
+    if ( ( dg.size() != g.size() ) )
     {
       throw BadProperty( "Conductances must have the same sizes." );
```

Comment:
I think this should be something like "There should be one time derivative entry for each conductance", or "The number of conductances and their first derivatives should be the same", but the arrays do not not both contain conductances.